src/main/java/nl/uu/cs/ape/domain/APEDomainSetup.java

```diff
@@ -551,4 +562,21 @@ public List<AuxiliaryPredicate> getHelperPredicates() {
         return helperPredicates;
     }
 
+    /**
+     * Write locally the SAT (CNF) workflow specification in human readable format.
+     * 
+     * @param satInputFile - File containing the SAT problem specification.
```

Comment:
The JavaDoc format for params here is not consistent with e.g. ConstraintFactory.java. Following that style, it would be e.g. "@param satInputFile    The file containing the SAT problem specification."

Reply:
Thanks for pointing it out, I will remove ` - ` to keep it consistent.